src/timepicker/timepicker.js

```diff
@@ -134,7 +134,7 @@ class TimePicker extends React.Component<TimePickerPropsT, TimePickerStateT> {
           ? undefined
           : {id: closestStep, label: secondsToLabel(closestStep)},
       });
-      this.handleChange(closestStep);
+      if (this.props.value) this.handleChange(closestStep);
```

Comment:
what should happen if there is no value, but nullable is false?

Reply:
Good point! Added an additional condition for it.